task/04-arrays-tasks.js

```diff
@@ -234,7 +234,13 @@ function toArrayOfSquares(arr) {
  *   [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ] => [ 1, 3, 6, 10, 15, 21, 28, 36, 45, 55 ]
  */
 function getMovingSum(arr) {
-   throw new Error('Not implemented');
+   const newArr = arr.map(function(x, i, arr){
+      if(i > 0){
+         arr[i] += arr[i - 1];
```

Comment:
Never change arguments. It is a potential bug. Why? External code will work with mutated array.